example/gradle.properties

```diff
@@ -0,0 +1 @@
+com.basistheory.android.example.apiUrl=https://api-dev.basistheory.com
```

Comment:
should we default to prod instead?

Reply:
🤷 I have this pointing to dev for the integration tests, which we typically run against dev. We could override it from the workflow back to dev. I'm going to leave it like this for now to avoid needing to change github secrets.